src/main/java/dev/monogon/cue/lang/psi/impl/CuePsiImplUtil.java

```diff
@@ -37,4 +42,45 @@ public static boolean isOptionalFieldName(@NotNull CueLabelName name) {
         var next = name.getNextSibling();
         return next != null && next.getNode().getElementType() == CueTypes.QMARK;
     }
+
+    static Logger logger = Logger.getInstance(CuePsiImplUtil.class);
+
+    public static ItemPresentation getPresentation(CueField field) {
```

Comment:
I prefer `@Nullable` and `@NotNull` annotations.
Please declare the return type as `@Nullable` and the paramter as `@NotNull`